Polyclinic/Polyclinic.API.Host/Controllers/AnalyticsController.cs

```diff
@@ -0,0 +1,64 @@
+using Microsoft.AspNetCore.Mvc;
+using Polyclinic.Application.Interfaces;
+using Polyclinic.Contracts.Dto;
+
+namespace Polyclinic.API.Host.Controllers;
+
+[ApiController]
+[Route("api/[controller]")]
+public class AnalyticsController(
+    IAnalyticsService analyticsService
+    ) : ControllerBase
+{
+
+
+    /// <summary>
+    /// Get doctors with experience more than specified years
+    /// </summary>
+    [HttpGet("doctors/experience/{years}")]
```

Comment:
```suggestion
    [HttpGet("doctors/experience/{years:int}")]
```
Лучше сразу указывать ожидаемый тип. Аналогично в других местах.